### DISCUSSION TITLE
setGravity from ammo-on-nodejs doesn't work

### DISCUSSION THREAD

**Discussion by ice-wizschool** [enable3d/enable3d#234] [Help / Questions] [answered]
I noticed that Physics from ammo-on-nodejs is extending AmmoPhysics. 
I'm trying to revise default gravity for this physics:

physics = new Physics();
physics.setGravity(0, -30, 0);

setGravity is working for AmmoPhysics, but it doesn't work no node.js headless mode. 

Please any tip for it?

**Comment by yandeu** [comment, marked as answer]
Isn't it like  
`new Physics({ gravity: { x: 0, y: -30, z: 0 })`?

**Reply by ice-wizschool** [reply]
thank you, @yandeu it works. I think it means we cannot change the gravity after initiating physics world right?

**Reply by ice-wizschool** [reply]
@yandeu And how can I reach you? my email address is ice@redbrick.space, can you please send your email address? I have some thoughts want to share with you. Thank you!

**Reply by yandeu** [reply]
> thank you, @yandeu it works. I think it means we cannot change the gravity after initiating physics world right?

Not sure, but you can change the gravity per body:
https://enable3d.io/examples/per-body-gravity.html

> @yandeu And how can I reach you? my email address is [ice@redbrick.space](mailto:ice@redbrick.space), can you please send your email address? I have some thoughts want to share with you. Thank you!

You will find my mail on my github page.
_I will not respond till mid January._
